src/graph/node.rs

```diff
@@ -309,6 +313,21 @@ impl Node {
                     mn.output_max = input_node.output_max / mult;
                 }
             }
+            OpKind::LeakyReLU(_) => {
+                let input_node = &inputs[0];
+                mn.in_dims = input_node.in_dims.clone();
+                mn.out_dims = input_node.out_dims.clone();
+                mn.output_max = input_node.output_max;
+                mn.in_scale = input_node.out_scale;
+                mn.out_scale = scale;
+                let scale_diff = mn.in_scale - mn.out_scale;
+                // We can also consider adjusting the scale of all inputs and the output in a more custom way.
+                if scale_diff > 0 {
+                    let mult = scale_to_multiplier(scale_diff);
+                    mn.opkind = OpKind::LeakyReLU(mult as usize); // now the input will be scaled down to match
+                    mn.output_max = input_node.output_max / mult;
```

Comment:
We also want to read in the leaky relu slope somewhere here; from the Onnx node.